pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -451,9 +460,14 @@ func buildTCPFilter(service *serviceMetadata, option rbacOption, is11 bool) *lis
 
 // buildHTTPFilter builds the RBAC http filter that enforces the access control to the specified
 // service which is co-located with the sidecar proxy.
-func buildHTTPFilter(service *serviceMetadata, option rbacOption, is11 bool) *http_conn.HttpFilter {
+func buildHTTPFilter(service *serviceMetadata, option rbacOption, is11 bool, isRbacV2 bool) *http_conn.HttpFilter {
 	option.forTCPFilter = false
-	config := convertRbacRulesToFilterConfig(service, option)
+	var config *http_config.RBAC
+	if isRbacV2 {
+		config = convertRbacRulesToFilterConfigV2(service, option)
+	} else {
+		config = convertRbacRulesToFilterConfig(service, option)
+	}
```

Comment:
Update the log in line 471 to include which version is used for generating the filter config. Same for the TCP case.

Reply:
Done. Thanks!